it-and-security/lib/macos-intel.queries.yml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: query
 spec:
-  name: query_apple_silicon
+  name: query_apple_intel
```

```suggestion
  name: Check if Intel-based Mac
```

What do you think about this name? As someone in dogfood I can tell what it's doing at a glance.

---

Also, based on this PR is looks like we don't need this `macos-intel.queries.yml` query anymore?

Instead we want the `macos-arm.queries.yml` query?

I could be misunderstanding :)

---

@noahtalerman If the convention allows for human-readable, cool. This does not follow our [GitOps query convention](https://github.com/fleetdm/fleet/blob/010bc54677082198b3a9bf9f44818f6177904e5b/docs/Using%20Fleet/GitOps.md?plain=1#L121-L140) so it does not actually show on the Queries page. 

There are no docs around this, and the example I was given was the following:
```yaml
apiVersion: v1
kind: query
spec:
  name: query_ruby
  query: select 1
```

And, yes, we probably don't need this, since we do not support platform-specific software versions.

---

> it does not actually show on the Queries page.

Ah, great point.

I think it still makes sense to use friendly names so when we, or a propsect looks at this 15 days from now, it's obvious what the query is doing.

I updated the query to `macos-check-if-apple-silicon`. What do you think?